.gitmodules

```diff
@@ -12,10 +9,10 @@
 	url = https://github.com/robojackets/rrt
 [submodule "external/grSim"]
 	path = external/grSim
-	url = https://github.com/RoboJackets/grSim
+	url = https://github.com/RoboCup-SSL/grSim
 [submodule "external/robocup-fshare"]
 	path = external/robocup-fshare
 	url = https://github.com/robojackets/robocup-fshare
 [submodule "external/sdlcontrollerdb"]
 	path = external/sdlcontrollerdb
-	url = https://github.com/kylestach/SDL_GameControllerDB
+	url = https://github.com/gabomdq/SDL_GameControllerDB
```

Comment:
can't actually test if this submodule works properly as I don't have a controller, but can revert if needed